pkg/pod/v1alpha1/pod.go

```diff
@@ -0,0 +1,111 @@
+package v1alpha1
+
+import v1 "k8s.io/api/core/v1"
+
+// pod holds the api's pod objects
+type pod struct {
+	object *v1.Pod
+}
+
+// PodList holds the list of pod instances
+type PodList struct {
```

Comment:
please recheck if we need to expose PodList struct
We might provide additional method on podList to avoid exposure of PodList & Items